.github/workflows/assign_by_path.yml

```diff
@@ -0,0 +1,28 @@
+name: assign_by_path
+
+on:
+  pull_request:
+    branches:
+      - 'pearson-release/*.master'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Assign by path
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Is this action required here? @Alec4r

Reply:
If in the future we need to make a custom model or changes in an e-commerce model will be necessary approval from the DevOps team.

Reply:
Ok, I understand that but, I was talking about the "actions/checkout@v2", is it necessary for this action to work? Can be removed? I think it's not necessary since the edx-platform action file does not use it... @Alec4r

Reply:
@Squirrel18 is necessary if not this will fail because doesn't find a "Git repository" to make the kdiff.

Reply:
@Squirrel18 yes, this will be added in the edx-platform PR too.

Reply:
Ok, thanks.